src/lib.rs

```diff
@@ -18,10 +22,8 @@ pub(crate) use visitor::Visitor;
 /// recursively inlined.
 ///
 /// This is equivalent to using an `InlinerBuilder` with the default settings.
-pub fn parse_and_inline_modules(src_file: &std::path::Path) -> syn::File {
-    InlinerBuilder::default()
-        .parse_and_inline_modules(src_file)
-        .unwrap()
+pub fn parse_and_inline_modules(src_file: &std::path::Path) -> Result<syn::File, Error> {
```

Comment:
Returning `syn::File` instead of a result was based on the signature from dtolnay/request-for-implementation#6. Since I operate on the belief that @dtolnay knows a lot more than me and has good reasons for most of his suggestions, how would you feel about leaving this signature as-is and adding `try_parse_and_inline_modules` or something as a sibling public function?

Reply:
Found the reason why this is infallible [here](https://github.com/dtolnay/request-for-implementation/issues/6#issuecomment-462453285).

I'm leaning towards leaving this function as-is, but updating its documentation and/or the crate documentation to make clearer that there's a way to build an inliner that will preserve/return errors found during inlining.